### DISCUSSION TITLE
Ability to change system tray icon

### DISCUSSION THREAD

**Discussion by MikaelAarras** [Flow-Launcher/Flow.Launcher#2458] [Ideas]
A feature that would allow system tray icon to be changed to a magnifying glass icon for example. Since the flow launcher already launches when clicking the system tray icon, it would make the feature much more pleasent to use.

**Comment by jjw24** [comment]
This level of customisation/personalisation is not something we would want to do, system tray icon uses the official flow logo and provides its identity. There are probably sys tray icon changers you can utilise instead.

**Reply by onesounds** [reply]
> This level of customisation/personalisation is not something we would want to do, system tray icon uses the official flow logo and provides its identity. There are probably sys tray icon changers you can utilise instead.

Let's consider putting this in.

It doesn't change the default app icon, so I think it's fine, and it tells you that it's searchable when you click it (of course, the default is the flow launcher icon, so it only makes sense if you enable the setting later).

Recently I got confused if it was the dev build I was using, so I thought of the option to change the tray icon to dev purple. I suppose we could reward people who have launched the flow launcher 500 times - if they've used it that many times, we can assume they've gotten used to it, so this kind of customization shouldn't affect the basic user too much.

```
if (flowLaunchCount > 500)
{ 
    trayiconselectoption.visibility = visibility.visible;
}
```
